olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -244,6 +245,11 @@ class DATASERVICE_READ_API VersionedLayerClient final {
   client::CancellableFuture<PrefetchTilesResponse> PrefetchTiles(
       PrefetchTilesRequest request);
 
+  client::CancellationToken GetData(TileRequest tile_request,
+                                    DataResponseCallback callback);
+
+  client::CancellableFuture<DataResponse> GetData(TileRequest tile_request);
+
```

Comment:
Please add proper Doxygen explaining why the users should use this API and what is the benefit using this instead of the DataRequest above.
Also please move these methods up so that all GetData() methods are together.